src/main/sensors/battery.c

```diff
@@ -329,10 +338,10 @@ static void batteryUpdateConsumptionState(void)
 void batteryUpdateCurrentMeter(timeUs_t currentTimeUs)
 {
     UNUSED(currentTimeUs);
-    if (batteryCellCount == 0) {
-        currentMeterReset(&currentMeter);
-        return;
-    }
+//    if (batteryCellCount == 0) {
+//        currentMeterReset(&currentMeter);
+//        return;
+//    }
```

Comment:
This was added as a fix for some boards / board & PDB combos that report a phantom current when the board is plugged into USB, but no battery is connected, in order to keep the consumption at 0 until a battery is plugged in. So the right thing to do here is to make this behaviour configurable.